app/app/src/main/kotlin/com/hedvig/android/app/ui/HedvigAppState.kt

```diff
@@ -110,7 +110,7 @@ internal class HedvigAppState(
       listOfNotNull(
         TopLevelGraph.HOME,
         TopLevelGraph.INSURANCE,
-        TopLevelGraph.REFERRALS.takeIf { isReferralsEnabled },
+        TopLevelGraph.FOREVER.takeIf { isReferralsEnabled },
```

Comment:
Might be able to rename the feature flag locally too, so it's `isForeverEnabled` and `featureManager.isFeatureEnabled(Feature.FOREVER)` but I know it's always hard to do renames, we'll definitely miss a thing or two here and there.